metricprofiler/src/main/java/com/expedia/adaptivealerting/metricprofiler/source/DefaultProfilingSource.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.metricprofiler.source;
+
+import lombok.RequiredArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+import java.util.Map;
+
+@RequiredArgsConstructor
+public class DefaultProfilingSource implements ProfilingSource {
+
+    @NotNull
+    private final ProfilingClient client;
+
+    @Override
+    public Boolean profilingExists(Map<String, String> tags) {
```

Comment:
Would be better to take a `MetricDefinition` parameter here, since that's the abstraction representing a metric.